esp/src/eclwatch/Utility.js

```diff
@@ -19,6 +20,13 @@
         ;
     }
 
+    function csvEncode(str) {
+        str = "" + str;
+        return str.replace(/"/g, '\"')
+                  .replace(/','/g, '\'')
+        ;
+    }
```

Comment:
If str is numeric it can be returned as is.
For our purposes we should always add quotes around strings.
All existing double quotes in the string should be escaped by adding an additional double quotes.

Some examples:

before|after
------|-----
123|123
abc|"abc"
abc"d"ef|"abc""d""ef"
abc'd'ef|"abc'd'ef"


```


should, it should not have quotes.
There is no need to escape the single quote instances
Double quotes are escaped 
To "escape"